ui/apps/platform/src/services/DelegatedRegistryConfigService.ts

```diff
@@ -0,0 +1,16 @@
+import axios from './instance';
+
+import { DelegatedRegistryConfig } from '../types/dedicatedRegistryConfig.proto';
+
+const delegatedRegistryUrl = '/v1/delegatedregistryconfig';
+
+/**
+ * Fetches the declarative config health objects.
+ */
+export function fetchDelegatedRegistryConfig(): Promise<{
```

Comment:
Sagas established wrapping pattern. What do you think about the following instead:
* return `Promise<DelegatedRegistryConfig>` here
* use `.then((response) => response.data)'` below
* be able to simplify call from component:

```js
.then((configFetched)) => {
    setDedicatedRegistryConfig(configFetched);
})
```